mysql-test/suite/rocksdb/t/i_s_trx_rpl.test

```diff
@@ -0,0 +1,42 @@
+--source include/master-slave.inc
+
+--disable_warnings
+DROP TABLE IF EXISTS t1;
+connection slave;
+--source include/stop_slave.inc
+--enable_warnings
+
+connection master;
+create table t1 (a int) engine=rocksdb;
+--disable_query_log
+--let $aa= 0
+while ($aa < 1000) {
+  eval insert into t1 values ($aa);
+  --inc $aa
+}
+--enable_query_log
+
+connection slave;
+show variables like 'rocksdb_rpl_skip_tx_api';
+--source include/start_slave.inc
+
+--let $it=0
+--let $stop=0
+while ($stop != 1) {
```

Comment:
I suspect this is going to be unstable without some sort of sync - what happens if slave fully syncs before this loop?

Also this burns CPU while waiting for the condition to become true, I'd add some --sleep 0.1 (and reduce $it limit accordingly)